modules/virt-configuring-rate-limiters.adoc

```diff
@@ -0,0 +1,18 @@
+
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-vm-control-plane-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-rate-limiters_{context}"]
+= Configuring rate limiters
+
+To compensate for large-scale burst rates, scale the `QPS`(Queries per Second) and `burst` `RateLimits` to process a higher rate of client requests or API calls concurrently for each component.
+
+Apply a jsonpatch annotation to adjust the `kubevirt-hyperconverged` cluster configuration using a `tuningPolicy` to apply scalable tuning parameters. This `tuningPolicy` will automatically adjust all virtualization components (`webhook`, `api`, `controller`, `handler`) to match the `QPS` and `burst` values specified by the profile.
+
++
```

Comment:
Did you want the note indented under the previous paragraph? If so then I think the line above the `+` needs to be removed.